python3-qtodotxt.json

```diff
@@ -0,0 +1,127 @@
+{
+  "app-id" : "org.qtodotxt.qtodotxt",
+  "runtime" : "org.kde.Platform",
+  "runtime-version" : "5.11",
+  "sdk" : "org.kde.Sdk",
+  "command" : "qtodotxt",
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=x11",
+    "--share=network",
```

Comment:
What does it need network access for?

Reply:
It does not need it i think , i removed it